cmd/swaprecover/main.go

```diff
@@ -15,12 +15,12 @@ import (
 	"github.com/athanorlabs/atomic-swap/cliutil"
 	"github.com/athanorlabs/atomic-swap/common"
 	mcrypto "github.com/athanorlabs/atomic-swap/crypto/monero"
+	contracts "github.com/athanorlabs/atomic-swap/ethereum"
```

Comment:
It would decrease code complexity if we didn't rename our own packages when importing. Is there a place where the generated code from contracts could be placed in a folder named `contracts`?